Immense.RemoteControl.Desktop.Shared/Services/DesktopHubConnection.cs

```diff
@@ -164,6 +165,12 @@ public async Task Disconnect()
         }
     }
 
+    public async Task Disconnect(string reason)
+    {
+        _logger.LogInformation("Disconnecting caster socket.  Reason: {reason}", reason);
```

Comment:
Weird tangent: You always use double spaces when starting a new sentence?  That's what I was taught to do, but Darren, Nick, Christian don't do it

Reply:
LOL!  Yeah, I was taught that way too, and now it's a habit I'll never break.